### ISSUE TITLE
Models

### ISSUE THREAD

**Issue by dkliachkovski** [dkliachkovski/AI_Captcha#6]
1. Extract content from an image using the 4_2 convolutional 
    layer of pre-trained vgg16/19

2. Obtain a style of an input image and use a constant as a 
    hyperparameter for changing style levels

3. Research the content loss and style loss
